templates/laravel/files/.lando.upstream.yml

```diff
@@ -0,0 +1,31 @@
+# This file sets some good defaults for local development using this Platform.sh
+# template with Lando.
+#
+# Note that you should not edit this file so it can continue to receive upstream
+# updates. If you wish to change the values below then override them in your
+# normal .lando.yml.
+
+# These both allow you to test this template without needing a site on Platform.sh
+# However you will want to replace them in your .lando.yml
+name: platformsh-laravel
+recipe: platformsh
+
+config:
+
+  # This section overrides Platform.sh configuration with values that make more
+  # sense for local development.
+  #
+  # Note that "app" is the name of the application defined in your
+  # .platform.app.yaml or applications.yaml.
+  overrides:
+    app:
+      variables:
+        env:
+          APP_ENV: development
```

Comment:
@Crell researching this a bit more i think i choose this because of the docs here https://github.com/platformsh/laravel-bridge#common-environment-variables-not-set, i took `development` to be a value as written. However, doing more research it looks like this is also often set to `local` (see https://github.com/platformsh-templates/laravel/blob/master/.env.example#L2).

Most important thing seems to be that it is just not set to `production`. I'm happy with either `dev`, `local` `development` or `lando`.